integration_tests/test_social_groups.py

```diff
@@ -0,0 +1,199 @@
+# -*- coding: utf-8 -*-
+def test_social_groups(session, login, codex_url):
+    # Create social group roles
+    login(session)
+    data = {
+        'key': 'social_group_roles',
+        'data': {
+            'Matriarch': {'multipleInGroup': False},
+            'Patriarch': {'multipleInGroup': True},
+        },
+    }
+    response = session.post(codex_url('/api/v1/site-settings/'), json=data)
+    assert response.status_code == 200
+    assert response.json() == {
+        'key': 'social_group_roles',
+        'data': {
+            'Matriarch': {'multipleInGroup': False},
+            'Patriarch': {'multipleInGroup': True},
+        },
+        'created': response.json()['created'],  # 2021-10-29T20:17:46.585288+00:00
+        'string': response.json()['string'],  # '' or None
+        'file_upload_guid': None,
+        'public': True,
+        'updated': response.json()['updated'],
+    }
+
+    # Create encounters
```

Comment:
I can see a lot of integration tests wanting to create a sighting and an individual. 
These should be utils for reuse

Reply:
:+1: We can move them to utils when there's another test that uses the same code